challengeutils/discussion.py

```diff
@@ -1,114 +1,193 @@
-import synapseclient
-import os
-from synapseclient.exceptions import *
-
+'''
+Functions that interact with the Synapse discussion API
+'''
+import json
+import requests
 QUERY_LIMIT = 1000
 
 
-def _getForumId(syn, synId):
-    return(syn.restGET('/project/%s/forum' % synId)['id'])
+def _get_forum_obj(syn, synid):
+    '''
+    Get forum id
+
+    Args:
+        syn: Synapse object
+        synid: Synapse Project id
```

Comment:
check what is common in client

should use `synapseclient.utils.id_of` to get the id.